SampleDestination/project.json

```diff
@@ -0,0 +1,50 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "version": "1.1.0-*",
+      "type": "platform"
+    },
+    "Microsoft.AspNetCore.Diagnostics": "1.2.0-*",
```

Comment:
Remove, this adds a new repo dependency.
http://aspnetci/viewLog.html?buildId=153650&buildTypeId=UniverseCoherence&tab=buildLog&state=248%2C240#_state=248,240&focus=570